manim/scene/scene_file_writer.py

```diff
@@ -567,37 +557,92 @@ def combine_files(
             for pf_path in input_files:
                 pf_path = Path(pf_path).as_posix()
                 fp.write(f"file 'file:{pf_path}'\n")
-        commands = [
-            config.ffmpeg_executable,
-            "-y",  # overwrite output file if it exists
-            "-f",
-            "concat",
-            "-safe",
-            "0",
-            "-i",
-            str(file_list),
-            "-loglevel",
-            config.ffmpeg_loglevel.lower(),
-            "-metadata",
-            f"comment=Rendered with Manim Community v{__version__}",
-            "-nostdin",
-        ]
 
+        av_options = {
+            "safe": "0",  # needed to read files
+        }
+
+        if not includes_sound:
+            av_options["an"] = "1"
+
+        partial_movies_input = av.open(
```

Comment:
Is `av.open` not used as a context manager here to save indentation?

Reply:
Yes, exactly.